OxfordBodleian/Juel-Jensen/BDLaethd16.xml

```diff
@@ -305,23 +305,351 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                     <q xml:lang="gez"><persName ref="PRS3926Eve">ሔዋን፡</persName></q>
+                                 </desc>
+                                </decoNote>
+
+                                <decoNote type="miniature" xml:id="d9">
+                                    <locus target="#12r"/>
+                                  <desc>Three episodes from top left to bottom right:
+                                    <persName ref="PRS2984Cain">Cain</persName> and
+                                    <persName ref="PRS1230Abel">Abel</persName> offer gifts to god (above);
+                                    <persName ref="PRS2984Cain">Cain</persName> kills
+                                    his brother (bottom left); he encounters a crow while wondering (bottom right).
+
+                                    <q xml:lang="gez"><persName ref="PRS1230Abel">አቤል፡</persName> / መሥዋዕት፡ / እሳት፡</q>
+                                    <q xml:lang="gez"><persName ref="PRS2984Cain">ቃየል</persName></q>
+                                    <q xml:lang="gez">ዘከመ፡ ቀተሎ፡ ቃየል፡ ለአቤል።</q>
+                                    <q xml:lang="gez"><persName ref="PRS1230Abel">አቤል፡</persName></q>
+                                    <q xml:lang="gez"><persName ref="PRS2984Cain">ቃየል</persName> / ቋዕ፡</q>
+                                </desc>
+                               </decoNote>
+
+                               <decoNote type="miniature" xml:id="d10">
+                                   <locus target="#12v"/>
+                                 <desc>God instructs <persName ref="PRS3814Enoch"/> on how to invoke his name as
+                                   <persName ref="PRS8782Seth"/> looks at the recumbent body of his dead brother
+                                   <persName ref="PRS1230Abel"/>.
+
+                                   <q xml:lang="gez">እግዚአብሔር።</q>
+                                    <q xml:lang="gez"><persName ref="PRS3814Enoch">ሄኖክ፡</persName></q>
+                                    <q xml:lang="gez"><persName ref="PRS8782Seth">ሴት፤</persName></q>
+                                    <q xml:lang="gez"><persName ref="PRS1230Abel">አቤል፡</persName></q>
+                               </desc>
+                               </decoNote>
+
+                               <decoNote type="miniature" xml:id="d11">
+                                   <locus target="#13r"/>
+                                 <desc><persName ref="PRS14010Jared">Jared</persName> and his children are shown above, while below the children of Seth descend
+                                   from a mountain to meet the wives of Cain, one of whom is a giant.
+                                    <!-- Had some trouble with the description of this scene can you make better sense of it? Does the upper scene
+                                  have something to do with the Dabra Mazagbet, I am unfamiliar with the place name -->
```

Comment:
I would translate it as the "Mount of the Treasures". It referes to the place known elsewhere as "Cave of the Treasures". This is the name of a Syriac work which later inspired two Arabic works, the Aksimaros and the "Life of Adam" (Sophia worked on them), which were both translated in Ethiopic and most probably are at the basis of this passage in the Ṭabiba ṭabibān.
The Cave (here Mount) of the Treasures is the legendary cave in which, according to some non-biblical traditions, Adam and Eve lived after being expelled from Paradise. It is so called because it was said to contain gold, myrrh and other "treasures". The first antediluvian patriarchs (presumably depicted in the scene) also lived there, until the sons of Seth decided to descend the mount (ወረዱ፡ እምደብር፡). In the biblical and non-biblical genealogies Jared is placed somewhere between Seth and Jared's much more famous son Enoch. By the way the Semitic root ወረደ፡ "descend" and Yāred is the same, which contributed to the legend.
So the caption ዘከመ፡ ዐቀቦ፡ ለያረድ፡ በደብረ፡ መዛግብት፡ means "How [God] watched over Jared in the Mount of the Treasures".

Reply:
Che bella spiegazione Max, grazie!